scripts/win32.bat

```diff
@@ -128,12 +129,14 @@ Function GetTopLevelDrives()
 			Summary.Add "Size", TopLevelDrive.Size
 
 			Set Mountpoints = new List
+			Set MountpointObjects = new List
```

Comment:
Probably yes, but I also don't know much VBScript.
There is this line in the code. I was not sure what it does
https://github.com/resin-io-modules/drivelist/blob/master/scripts/win32.bat#L135

`Mountpoints` is a list of strings and I introduced `MountpointObjects` which is a list of objects. But list function works only with simple types:
https://github.com/resin-io-modules/drivelist/blob/master/scripts/win32.bat#L37

So I used the `Mountpoints` for condition check, but store actual data inside `MountpointObjects`.

Not the best solution, imho, but was a minimal change in the code